Kconfig

```
@@ -9,6 +9,19 @@ config APPSDIR
 	string
 	option env="APPSDIR"
 
+menu "License Setup"
+config ALLOW_GPL_COMPONENTS
+	bool "Use components that have GPL/LGPL licenses"
+	default n
+	---help---
+		When this option is enabled the project will use components
```

```suggestion
		When this option is enabled the project will allow the use of components
```
"The project will use" is a bit misleading, since the new allowed components may not be auto-selected.